stock_prices/stock_prices.py

```diff
@@ -3,7 +3,11 @@
 import argparse
 
 def find_max_profit(prices):
-  pass
+  profits = []
+  for i in range (0, len(prices)):
```

Comment:
Good job solving this. You can solve it without the double for loop to lower time complexity by keeping track of your max profit while iterating through the first for loop.

min_price = prices[0]
max_profit = prices[1] - min_price

for i in range(1, len(prices)):
____price = prices[i]
____max_profit = max(price - min_price, max_profit)
____min_price = min(price, min_price)

return max_profit